mobile/src/calendar/days-counters/days-counters.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { View, Text, LayoutChangeEvent, ViewStyle, StyleSheet } from 'react-native';
+import { daysCountersStyles } from '../styles';
+import { DaysCounter, EmptyDaysCounter } from './days-counter';
+import { DaysCounterSeparator } from './days-counter-separator';
+import { DaysCountersModel } from '../../reducers/calendar/days-counters.model';
+import { AppState } from '../../reducers/app.reducer';
+import { connect, Dispatch } from 'react-redux';
+import { Employee } from '../../reducers/organization/employee.model';
+import { calculateDaysCounters, CalendarActions } from '../../reducers/calendar/calendar.action';
+import { SelectedDay } from './selected-day';
+import { Triangle } from './triangle';
+import moment from 'moment';
+
+interface DaysCountersProps {
+    employee: Employee;
+    daysCounters: DaysCountersModel;
+}
+
+interface DaysCountersDispatchProps {
+    calculateDaysCounters: (vacationDaysLeft: number, hoursCredit: number) => void;
+}
+
+interface DaysCounterState {
+    daysCountersWidth: number;
+}
+
+class DaysCountersImpl extends Component<DaysCountersProps & DaysCountersDispatchProps, DaysCounterState> {
+    constructor(props: DaysCountersProps & DaysCountersDispatchProps) {
+        super(props);
+        this.state = {
+            daysCountersWidth: 0
+        };
+    }
+
+    public componentWillReceiveProps(nextProps: Readonly<DaysCountersProps>, nextContext: any) {
+        const { employee } = nextProps;
+        if (this.props.employee !== employee) {
+            this.props.calculateDaysCounters(employee.vacationDaysLeft, employee.hoursCredit);
+        }
+    }
+
+    public render() {
+        const { daysCounters: { allVacationDays, hoursCredit } } = this.props;
+
+        const vacationCounter = allVacationDays
+            ? <DaysCounter  textValue={allVacationDays.toString()}
+                            title={allVacationDays.title} />
+            : <EmptyDaysCounter />;
+
+        const daysoffCounter = hoursCredit
+            ? <DaysCounter  textValue={hoursCredit.toString()}
+                            title={hoursCredit.title} />
+            : <EmptyDaysCounter />;
+
+        // TODO: temp
```

Comment:
please edit a comment to indicate what needs to be changed

Reply:
fixed